azure-pipelines-deploy-template.yml

```diff
@@ -100,6 +100,14 @@ jobs:
                 $encodedAccessToken = [System.Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($env:SYSTEM_ACCESSTOKEN))
                 $variableGroupLastEdit = @()
 
+                #Check if RG exists
+                Get-AzResourceGroup -Name $resourceGroupName -ErrorVariable rgDoesntExist -ErrorAction SilentlyContinue
+                if($rgDoesntExist){
+                  $fullDeployment = $true
+                  Write-Host "##vso[task.setvariable variable=fullDeployment]$fullDeployment"
```

Comment:
I'd delete this line and the following one and then enclose all of the following code in an `else`, except for the last line.
I'm not a huge fan of exiting from a script halfway through if I can avoid it, but sometimes it is unavoidable. In this case it avoids unnecessary code duplication.

Reply:
changed as suggested 👍